client/src/App.js

```diff
@@ -30,8 +32,8 @@ function App() {
             <Route exact path="/profiles/" component={ProfileListings} />
             <Route path="/profiles/:id" component={ProfileDetails} />
             <PrivateRoute exact path="/become-a-sitter" component={BecomeASitter} />
-            <PrivateRoute exact path="/my-sitters" />
-            <PrivateRoute exact path="/my-jobs" />
+            <PrivateRoute exact path="/my-sitters" component={MySitters} />
```

Comment:
Maybe can consider https://reactrouter.com/web/api/Route/render-func with isMyJobs prop and avoid the extra nesting of components

Reply:
I'll take a look at that yeah, it would make it a lot more straightforward

Reply:
I can't figure out why but `render={() => <Bookings/>}` is not working, the screen just shows up blank. I can't find any info on it online

Reply:
Left it as is for now